docs/src/guides/typo3/deploy/deploy.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Deploy TYPO3"
+sidebarTitle: "Deploy"
+weight: -80
+toc: false
+description: |
+    Now that your site is ready, push it to Platform.sh and import your data.
+---
+
+## Deployment
+
+If you have been customizing your existing project as you go, now is the time to ensure all code is committed to Git and to `git push` to Platform.sh on the `master` branch.  That will cause Platform.sh to build your code, producing a read-only file system image of your application, and then deploy it into a running cluster of containers.
+
+You can view the process from the Management Console; when it is done, click the URL shown in the Console to see your site.
+
+## Post-install (new site)
```

Comment:
Remove this. Will not be good for your database if you're starting with an existing site